llvm/lib/CodeGen/SelectionDAG/LegalizeDAG.cpp

```diff
@@ -2205,7 +2205,13 @@ void SelectionDAGLegalize::ExpandFPLibCall(SDNode* Node,
     Results.push_back(Tmp.first);
     Results.push_back(Tmp.second);
   } else {
-    SDValue Tmp = ExpandLibCall(LC, Node, false).first;
+    bool IsSignedArgument = false;
+    switch (Node->getOpcode()) {
+    case ISD::FLDEXP:
+      IsSignedArgument = true;
+      break;
+    }
```

Comment:
```suggestion
    bool IsSignedArgument = Node->getOpcode() == ISD::FLDEXP;
```

Reply:
I used the original structure to make it more obvious that there might be other (future) instructions that also need signed arguments.  That said, if this is the only barrier to merge I'll accept the change.

Is there a timeline on when this PR might be merged?  I'm coordinating with other Debian maintainers at the moment, as we need to not only stable-update Clang but also rebuild affected packages.

Thanks!

Reply:
No need to make it complicated for one case; if you wanted a general thing it should be a separate helper function

Reply:
Done.